packages/codemods/src/codemods-helpers.tsx

```diff
@@ -10,11 +11,59 @@ import {
   Node,
   ImportNamespaceSpecifier,
   Collection,
+  FileInfo,
+  API,
+  Options,
 } from 'jscodeshift';
+
 import { COMPILED_IMPORT_PATH, REACT_IMPORT_PATH, REACT_IMPORT_NAME } from './constants';
+import type { CodemodPlugin } from './plugins/types';
+import DefaultPlugin from './plugins/default';
 
 type Identifiers = Array<Identifier | JSXIdentifier | TSTypeParameter>;
 
+const getPlugins = async (
+  pluginPathsInput: string | Array<string> | undefined
+): Promise<Array<CodemodPlugin>> => {
+  if (!pluginPathsInput) return [];
+  const pluginPaths = Array.isArray(pluginPathsInput) ? pluginPathsInput : [pluginPathsInput];
+
+  return Promise.all(
+    pluginPaths.map(async (path) => {
+      try {
+        const pluginModule = await import(path);
+
+        const pluginName = pluginModule?.default?.metadata?.name ?? null;
+        if (pluginName === null) {
```

Comment:
Can you just have `!pluginName` to also check for `undefined` and empty strings, then omit `?? null`?